design/one-pager-stack-relationship-labels.md

```diff
@@ -0,0 +1,166 @@
+# Stack Parent/Child Relationship Label Spec
+
+- Owner: Steven Rathbauer (@rathpc)
+- Reviewers: Crossplane Maintainers
+- Status: Draft
+
+## Proposal
+
+[crossplaneio/crossplane#934](https://github.com/crossplaneio/crossplane/issues/934)
+
+## Problem
+
+There is currently no easy way to get a list of all resources that are owned by a parent resource.
+We can leverage CRD labels to make this easier. As long as we have a standard format for doing so.
+
+Per the proposal, it seems that the closest example of what I am referring to is shown on 
+[this page](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels) 
+within the `app.kubernetes.io/part-of` label. However since we are not exactly using these labels 
+for the intentions defined within that page, we should use something following the patterns instead.
+
+As long as we define this label on all children CRD's, we should then be able to query all children
+of a given parent object using those labels.
+
+## Design
+
+Wordpress example file tree structure of the example files below:
```

Comment:
We are presenting this as information that is provided and known statically, at build/package time.  When a stack author is building their stack package, they won't include all this metadata in statically within their package, right?  This is information that will need to be applied dynamically at runtime by a controller.  Do you think it would be less confusing to not mention the "file tree structure" or ".yaml files" here to avoid confusion?  It may be easier to convey this as a runtime resource graph or list, instead of a build time stack package filesystem layout.  What do you think?

Reply:
By the way, I don't think you need to get into the details of how a controller specifies these labels at runtime, that's probably out of scope for your efforts in this 1-pager that is simply trying to capture the experience and details of the metadata we'll use.  But I do think it's worth mentioning that a controller will be applying the labels at run time as opposed to the stack author at build time.

Reply:
@jbw976 So the initial reason I had added all of these file examples was to provide a clear example of what these would look like after. I found that within some of the previous design docs, there was a divergence in continuity between one example file to the next so there wasn't a way to construct a full picture of the example.

I can certainly add some clarification that these are post processed yaml outputs and not static package files.

Reply:
Sounds good, I think that clarification would help here.

Reply:
adding now